Dockerfile

```diff
@@ -26,58 +26,49 @@ ARG RVM_VERSION=stable
 ENV RVM_VERSION=${RVM_VERSION}
 
 # RMV user to create
+# Optional: child images can change to this user, or add 'rvm' group to other user
 ARG RVM_USER=rvm
 ENV RVM_USER=${RVM_USER}
 
-# Install RVM
-RUN apt update \
-    && apt install -y \
-       curl \
-       git \
-       gnupg2 \
+# Install RVM dependencies
+RUN sed -i 's~http://archive\(\.ubuntu\.com\)/ubuntu/~mirror://mirrors\1/mirrors.txt~g' /etc/apt/sources.list \
+    && apt-get update -qq \
+    && apt-get install -qy --no-install-recommends \
+       ca-certificates curl netbase \
+       git openssh-client \
+       gnupg dirmngr \
+       patch make gawk \
     && rm -rf /var/lib/apt/lists/*
 
 # Install + verify RVM with gpg (https://rvm.io/rvm/security)
-RUN gpg2 --quiet --no-tty --logger-fd 1 --keyserver hkp://keys.gnupg.net \
-         --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
-                     7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
-    && echo 409B6B1796C275462A1703113804BB82D39DC0E3:6: | \
-       gpg2 --quiet --no-tty --logger-fd 1 --import-ownertrust \
-    && curl -sSO https://raw.githubusercontent.com/rvm/rvm/${RVM_VERSION}/binscripts/rvm-installer \
-    && curl -sSO https://raw.githubusercontent.com/rvm/rvm/${RVM_VERSION}/binscripts/rvm-installer.asc \
-    && gpg2 --quiet --no-tty --logger-fd 1 --verify rvm-installer.asc \
-    && bash rvm-installer ${RVM_VERSION} \
-    && rm rvm-installer rvm-installer.asc \
-    && echo "bundler" >> /usr/local/rvm/gemsets/global.gems \
+RUN mkdir ~/.gnupg && chmod 700 ~/.gnupg && echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf \
+    && gpg --quiet --no-tty --keyserver hkp://keys.gnupg.net \
+           --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+    && ( curl -sSL https://get.rvm.io | bash -s ${RVM_VERSION} ) \
```

Comment:
Why did you choose to remove the lines from [Run verified installation](https://rvm.io/rvm/security#run-verified-installation) here? Those lines were the officially "most secure and verified" way to install RVM, right?

Reply:
The latest official installing scripts are:
```bash
gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
curl -sSL https://get.rvm.io | bash -s stable
```

Refer: https://rvm.io/rvm/install

Reply:
Hi @nthachus I'm not sure that we're reading that section the same. Particularly in a heavily repeated process such as building a Docker image, I would think you would _"...want to ensure the installer comes from a valid source, you can run manually verified installation:"_ as it says in the second section.

Can you say a little more about why you don't think building a Docker image is a case where we "want to ensure the installer comes from a valid source"?